.github/workflows/sozisel.yml

```diff
@@ -0,0 +1,69 @@
+name: Sozisel CI
+
+on:
+  pull_request:
```

Comment:
How does that work, every commit that is pushed to a pull request that points to master will trigger this pipeline?

Reply:
yes, exactly.